src/encdef.ts

```diff
@@ -77,15 +72,16 @@ vlEncDef.fromShorthand = function(shorthand) {
     }
   }
 
+  // TODO
```

Comment:
@kanitw Cannot compile with this code because we reference `schema.timefns`. One bug discovered by ts :-)